now.json

```diff
@@ -0,0 +1,15 @@
+{"version": 2,
+  "builds": [
+    {
+      "src": "package.json",
+      "use": "@now/static-build",
+      "config": {
+        "distDir": "out"
+      }
+    }
+  ],
+  "routes": [
+    { "handle": "filesystem" },
+    { "src": "/.*", "status": 404, "dest": "/404Error" }
+  ]
+}
```

Comment:
I think this is only relevant if you deploy on Zeit... which we don't.
Does this actually work / do anything?